app/src/main/java/com/example/homework_04032019/MainActivity.java

```diff
@@ -1,13 +1,57 @@
 package com.example.homework_04032019;
 
-import android.support.v7.app.AppCompatActivity;
+import android.content.Intent;
+import android.content.IntentFilter;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
+    public static final String CURRENT_COUNTER_TEXT = "currentCounterText";
+    private Intent intentAnyIntentService;
+    private AnyBroadcastReceiver anyBroadcastReceiver;
+    private TextView counterTextView;
+    private Button startServiceButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        counterTextView = findViewById(R.id.counterTextView);
```

Comment:
please don't forget and use clean code knowledge. Eg. this method will look much better if split it to few separate